manifest-build-tools/application/update_manifest.py

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/env python
+
+# Copyright 2015, EMC, Inc.
+
+
+import argparse
+import datetime
+import json
+import os
+import re
+import subprocess
+import sys
+import tempfile
+import shutil
+
+#pylint: disable=relative-import
+import config
+from RepositoryOperator import RepoOperator
+from manifest import Manifest
+
+
+class UpdateManifest(object):
+
+    def __init__(self):
```

Comment:
minor suggestions:
```
args_list = split_args(sys.argv[1:])
update = UpdateManifest()
```
looks all good to me,
In addition, if ```__init__``` can be 
```
def __init__(self, arg_list, arg1, arg2, arg3, arg4......):
```
to support initialization when statement: ```update = UpdateManifest(...)```
may be better for invoking.